tests-e2e/fixture/binding/fixture.go

```diff
@@ -105,6 +105,81 @@ func HaveStatusGitOpsDeployments(gitOpsDeployments []appstudiosharedv1.BindingSt
 	}, BeTrue())
 }
 
+func HaveGitOpsDeploymentsWithStatusProperties(gitOpsDeployments []appstudiosharedv1.BindingStatusGitOpsDeployment) matcher.GomegaMatcher {
+
+	// compare compares two slices, returning true if the fields other than the CommitID are equal, regardless of the order of elements in the slices
```

Comment:
FYI, in some cases we may be able to use `ContainElements` function from Gomega (e.g. `Expect(a).ToContainElements(b)`). This function will compare the list of elements, and ignore the order,  instead of writing a `compare` function. BUT because the commit id is non-deterministic, what we have here works for me.

Reply:
I've looked at the ContainElements function and will keep that in mind.

"ContainElements succeeds if actual contains the passed in elements. The ordering of the elements does not matter. By default ContainElements() uses Equal() to match the elements, however custom matchers can be passed in instead."